subprojects/composite-builds/build.gradle.kts

```diff
@@ -15,6 +15,7 @@ dependencies {
     implementation(libs.slf4jApi)
     implementation(libs.guava)
 
+    testImplementation(project(":file-watching"))
```

Comment:
Can you remove this dependency again, given that the enum isn't an inner class any more?

Reply:
Can't:

```Execution failed for task ':composite-builds:compileTestGroovy'.
> org.gradle.internal.watch.vfs.WatchMode
```